source/compatibility.txt

```diff
@@ -42,11 +42,11 @@ including the following:
 - Microsoft Azure Event Hubs
 - Red Hat AMQ Streams
 
-The {+connector+} works directly with {+kafka-connect-long+}. The connector does not
+The connector works directly with {+kafka-connect+}. The connector does not
```

Comment:
Just to clarify, you can continue to use the "{+connector+}" source constant here since it's not the full name, but it isn't necessary in the context of this page. No changes needed.